spec/classes/github_actions_runner_spec.rb

```diff
@@ -6,38 +6,29 @@
       let(:facts) { os_facts }
       let(:params) do
         {
-          :ensure                => 'present',
-          :org_name              => 'test_org',
-          :personal_access_token => 'test_PAT',
-          :package_name          => 'test_package',
-          :package_ensure        => '1.0.1',
-          :repository_url        => 'https://test_url',
-          :user                  => 'test_user',
-          :group                 => 'test_group',
-          :base_dir_name         => '/tmp/actions-runner',
-          :instances             => { 'first_runner' => { 'labels' => ['test_label1', 'test_label2'], 'repo_name' => 'test_repo'}},        }
```

Comment:
I miss tests for some of these parameters:
- org_name
- personal_access_token
- package_name
- package_ensure
- repository_url, and
- base_dir_name

Add another test setting those, please

Reply:
yes - because originally those were cover by default test , now im using the default values from the module hiera to test the manifest .

since those value already work with the same test i don't see the added value of repeating the same tests with different values . maybe it made since for user and group of the files . since we added new "exec"